pingdom-checks/clinical-fhir.ping

```diff
@@ -4,51 +4,51 @@ CLINICAL_FHIR_SLACK_ID="${SLACK_VASDVP_MONITORING},${SLACK_API_ALARMS_CLINICAL_F
 # Sandbox
 #
 
-# Nginx Proxy Health
+# Nginx Proxy Health (Liveness)
 pingdom save-check \
   --template https-public-200 \
-  -a name=sandbox-clinical-fhir-health \
+  -a name=sandbox-clinical-fhir-liveness \
```

Comment:
After discussing this with @jhulbert-va , let's avoid changes to the name field. Updating the url field alone should be the goal.

Reply:
Agreed. The pingdom script will update the check based on the name field. This means that changing it will create an entirely new health-check but won't remove the old one, therefore it should not be changed.